activesupport/lib/active_support/process_time.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module ProcessTime # :nodoc:
+    class << self
+      def clock_monotonic
+        Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      end
+
+      if defined?(Process::CLOCK_PROCESS_CPUTIME_ID) && !Gem.win_platform?
+        def clock_process_cputime_id
```

Comment:
This doesn't return `Process::CLOCK_PROCESS_CPUTIME_ID`, so I'd rather name this something else. Maybe `process_time`? I think [this](https://stackoverflow.com/a/7506987) breaks down the difference between constants correctly.